tasks/common.yml

```diff
@@ -52,3 +52,8 @@
     line: "rotate {{ log_retention_week }}"
     state: present
   when: log_retention_week is defined
+
+- name: Set http_common
+  set_fact:
+    httpd_common: True
```

Comment:
I would rather use a variable prefixed by _, to denote that's internal. Current code base is a bit inconsistant with that, but I rather not augment entropy.